spec/features/attending_events_spec.rb

```diff
@@ -53,6 +53,7 @@
     expect(page.current_path).to eq "/tickets"
 
     visit print_path(25)
+    sleep 2
```

Comment:
Are these absolutely necessary? why? using `sleep` to manipulate test should be avoided.